front/app/containers/Admin/projects/routes.tsx

```diff
@@ -45,157 +73,138 @@ const createAdminProjectsRoutes = () => {
               </LoadingComponent>
             ),
           },
+          {
+            path: 'edit',
+            element: (
+              <LoadingComponent>
+                <AdminProjectsProjectIndex />
+              </LoadingComponent>
+            ),
+          },
+          {
+            path: 'timeline',
+            element: (
+              <LoadingComponent>
+                <AdminProjectTimeline />
+              </LoadingComponent>
+            ),
+          },
+          {
+            path: 'timeline/new',
+            element: (
+              <LoadingComponent>
+                <AdminProjectTimelineNewAndEdit />
+              </LoadingComponent>
+            ),
+          },
+          {
+            path: 'timeline/:id',
+            element: (
+              <LoadingComponent>
+                <AdminProjectTimelineNewAndEdit />
+              </LoadingComponent>
+            ),
+          },
+          {
+            path: 'events',
+            element: (
+              <LoadingComponent>
+                <AdminProjectEvents />
+              </LoadingComponent>
+            ),
+          },
+          {
+            path: 'events/new',
+            element: (
+              <LoadingComponent>
+                <AdminProjectEventsEdit />
+              </LoadingComponent>
+            ),
+          },
+          {
+            path: 'events/:id',
+            element: (
+              <LoadingComponent>
+                <AdminProjectEventsEdit />
+              </LoadingComponent>
+            ),
+          },
+          {
+            path: 'permissions',
+            element: (
+              <LoadingComponent>
+                <AdminProjectPermissions />
+              </LoadingComponent>
+            ),
+          },
+          {
+            path: 'survey-results',
+            element: (
+              <LoadingComponent>
+                <AdminProjectSurveyResults />
+              </LoadingComponent>
+            ),
+          },
+          {
+            path: 'poll',
+            element: (
+              <LoadingComponent>
+                <AdminProjectPoll />
+              </LoadingComponent>
+            ),
+          },
+          // double check on this to make sure it goes in the right spot with regard to children/parent routes
```

Comment:
Checked? :)

Reply:
This is addressed in the other PR (which includes the dynamic module routing) but right now the project edit page is not responding (https://citizenlab.atlassian.net/browse/CL-848) so I can't test it directly